### PR TITLE
Add instructions for HTTP redirecting

### PR DESCRIPTION
**Pull request by SimonFrings**
This PR builds on top of #13 and #114.

### CODE REVIEW

File: docs/api/response.md [hunk 1]
```diff
@@ -263,6 +263,36 @@ Hello wörld!
...
+```
+
+Redirect responses have a status code in the 3xx range, and a [`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)
+response header holding the URL to redirect to. When a Browser or a search engine
+crawler receives a redirect, they will automatically follow the new URL provided
+in the `Location` header.
+
+See [response status codes](#status-codes) for more details.
+
+You can also use the [redirect helper method](https://framework-x.org/docs/api/app/#redirects)
```
Comment (clue) [inline docs/api/response.md:293]: Looks like this should be a relative link instead?

File: docs/api/response.md [hunk 2]
```diff
@@ -263,6 +263,36 @@ Hello wörld!
...
+    return new React\Http\Message\Response(
+        React\Http\Message\Response::STATUS_FOUND,
+        [
+            'Location' => '/blog'
+        ]
+    );
+});
+```
+
+Redirect responses have a status code in the 3xx range, and a [`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)
```
Comment (clue) [inline docs/api/response.md:286]:
```suggestion
Redirect responses have a status code in the `3xx` range, and a [`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)
```

File: docs/api/response.md [hunk 3]
```diff
@@ -263,6 +263,36 @@ Hello wörld!
...
+response header holding the URL to redirect to. When a Browser or a search engine
+crawler receives a redirect, they will automatically follow the new URL provided
+in the `Location` header.
+
+See [response status codes](#status-codes) for more details.
+
+You can also use the [redirect helper method](https://framework-x.org/docs/api/app/#redirects)
+for simpler use cases.
```
Comment (clue) [inline docs/api/response.md:294]: Does it make sense to include an example here?

File: docs/api/response.md [hunk 4]
```diff
@@ -263,6 +263,36 @@ Hello wörld!
...
+    );
+});
+```
+
+Redirect responses have a status code in the 3xx range, and a [`Location`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)
+response header holding the URL to redirect to. When a Browser or a search engine
+crawler receives a redirect, they will automatically follow the new URL provided
+in the `Location` header.
+
+See [response status codes](#status-codes) for more details.
```
Comment (clue) [inline docs/api/response.md:291]: Definitely a useful link, but perhaps we can make this more useful by providing more context, e.g. a list of common status codes for redirects?

**Review by clue — COMMENTED** [review]
@SimonFrings Thank you for looking into this, this looks like a nice addition! :+1:

File: docs/api/response.md (hunk 2 above)
Comment (SimonFrings) [reply docs/api/response.md]: :+1:

File: docs/api/response.md (hunk 3 above)
Comment (SimonFrings) [reply docs/api/response.md]: Yes, I think this is helpful :+1:

File: docs/api/response.md (hunk 4 above)
Comment (SimonFrings) [reply docs/api/response.md:299]: I added a list with commonly used redirect status codes, WDYT?

**Review by clue — APPROVED** [review]
@SimonFrings Thank you for the update, keep it up! :shipit:

**Pull request merged**